bin/rucio

```diff
@@ -229,20 +229,20 @@ def exception_handler(function):
             else:
                 logger.error(error)
                 logger.debug(traceback.format_exc())
-                logger.error("This means that the object doesn't have the property " + str(error) +
-                             ". This should never happen. Please rerun the last command with the '-v'" +
-                             " option and submit a ticket with all the necessary information at https://its.cern.ch/jira/browse/RUCIO")
-            return FAILURE
+                logger.error('The object is missing this property: ' + str(error))
+                logger.error('This should never happen. Please rerun the last command with the "-v" option to gather more information.')
+                logger.error('Please followup with all relevant information at: %s' % config_get('policy', 'support_email'))
+            raise
```

Comment:
just for info, it's recommended to split one string on several lines like that
```
s = ("this is a very"
      "long string too"
      "for sure ..."
     )
```
here:

```
logger.error('The object is missing this property: ' + str(error)
                     'logger.error('This should never happen. Please rerun the last command with the "-v" option to gather more information.')
```